tutorials/io/data_paths.rst

```diff
@@ -37,8 +37,8 @@ when the game is running, the project's file system will likely be read-only.
 
 The ``user://`` prefix points to a different directory on the user's device. On
 mobile and consoles, this path is unique to the project. On desktop, the engine
-stores user files in ``~/.local/share/godot/app_userdata/Name`` on macOS and
-Linux, and ``%APPDATA%/Name`` on Windows. ``Name`` is based on the application
+stores user files in ``~/.local/share/godot/app_userdata/Name`` on
+Linux, ``Library/Application\ Support/Godot/app_userdata`` on Mac OSX (since Catalina) and ``%APPDATA%/Name`` on Windows. ``Name`` is based on the application
```

Comment:
The backslash isn't part of the path – it's used to escape the space in shell when the path isn't quoted. Also, you've specified a relative path, but all paths should be absolute:

```suggestion
Linux, ``~/Library/Application Support/Godot/app_userdata`` on macOS (since Catalina) and ``%APPDATA%/Name`` on Windows. ``Name`` is based on the application
```

Reply:
Agreed, thank you.